modules/retail/pages/retail-best-practices.adoc

```diff
@@ -1,75 +1,64 @@
 [[retail-best-practices]]
-= Best Practices
+= Best practices
 
-== Deploying New Images
+== Deploying new images
 
 This file describes how to deploy a new image on selected terminals first, test it, and then deploy it to all terminals.
 
-The hardware type (HWTYPE) group is used for booting new terminals, while already deployed terminals can
-be moved to any group that provides the {saltboot} formula. 
-This flexibility allows for testing of new images
-or configurations. 
-To ensure a smooth deployment, follow these steps:
+The hardware type group [literal]``HWTYPE`` is used for booting new terminals, while already deployed terminals can be moved to any group that provides the {saltboot} formula. 
+This flexibility allows for testing of new images or configurations. 
 
-* Build New Image
-
-  * Use Image Sync formula to synchronize the image to all Branch Servers
-
-* Create a Test Group
+//To ensure a smooth deployment, follow these steps:
+.Procedure: Smooth image deployment
```

Comment:
Smooth is interesting choice 😃 

This procedure is how one can have two groups controlling the image deployment. In practice this is mimicking A/B image deployment. So we can call this either `A/B image deployment`, or `Dual group image deployment`.